server/sonar-server/src/main/java/org/sonar/server/issue/Action.java

```diff
@@ -78,6 +78,8 @@ public boolean supports(DefaultIssue issue) {
     DefaultIssue issue();
 
     IssueChangeContext issueChangeContext();
+
+    String issueOrganizationUuid();
```

Comment:
DefaultIssue is too much complex and we want to get rid of it.
Adding the organizaiton uuid here is a quick win that not brings complexity and that not impact usages of DefaultIssue.
Action is only used for bulk change so IMO it's not at all a pb.